static/app/views/replays/detail/userReplays/index.tsx

```diff
@@ -0,0 +1,209 @@
+import {useRef} from 'react';
+import {
+  AutoSizer,
+  CellMeasurer,
+  GridCellProps,
+  InfiniteLoader,
+  MultiGrid,
+} from 'react-virtualized';
+import styled from '@emotion/styled';
+
+import Placeholder from 'sentry/components/placeholder';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useRoutes} from 'sentry/utils/useRoutes';
+import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
+import NoRowRenderer from 'sentry/views/replays/detail/noRowRenderer';
+import useInfiniteLoader, {
+  LoadingStatus,
+} from 'sentry/views/replays/detail/useInfiniteLoader';
+import useLoadReplaysFromUser from 'sentry/views/replays/detail/userReplays/useLoadReplaysFromUser';
+import useVirtualizedGrid from 'sentry/views/replays/detail/useVirtualizedGrid';
+import HeaderCell from 'sentry/views/replays/replayTable/headerCell';
+import TableCell from 'sentry/views/replays/replayTable/tableCell';
+import {ReplayColumns} from 'sentry/views/replays/replayTable/types';
+import type {ReplayListRecord, ReplayRecord} from 'sentry/views/replays/types';
+
+type Props = {
+  replayId: undefined | string;
+  user: undefined | ReplayRecord['user'];
+};
+
+const COLUMNS = [
+  ReplayColumns.user,
```

Comment:
This column isn't the best; it's really repetitive to see the username over and over again. 

Maybe if it was just an icon, or if the icon was at the top and then this was something else... but right now it's our link into the replay that the row represents.